crate/get-help.md

```diff
@@ -1 +1 @@
-[project documentation](https://crate.io/docs/), [StackOverflow](https://stackoverflow.com/tags/crate), [Slack](https://crate.io/docs/support/slackin/), or [paid support](https://crate.io/pricing/)
+[project documentation](https://crate.io/docs/), [StackOverflow](https://stackoverflow.com/tags/crate), [support channels](https://crate.io/support/)
```

Comment:
In order to align with https://github.com/crate/crate-admin/pull/702, I am humbly asking to update the tag on the link to Stack Overflow.

```suggestion
[project documentation](https://crate.io/docs/), [StackOverflow](https://stackoverflow.com/tags/cratedb), [support channels](https://crate.io/support/)
```